ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    print(metadata)
</pre><p>can we save the output here into the notebook so it can be viewed on GitHub?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>